webui/react/src/pages/ModelDetails.tsx

```diff
@@ -32,13 +32,13 @@ import settingsConfig, { Settings } from './ModelDetails/ModelDetails.settings';
 import ModelHeader from './ModelDetails/ModelHeader';
 
 interface Params {
-  modelId: string;
+  modelName: string;
 }
 
 const ModelDetails: React.FC = () => {
   const { auth: { user } } = useStore();
   const [ model, setModel ] = useState<ModelVersions>();
-  const { modelId } = useParams<Params>();
+  const modelName = decodeURIComponent(useParams<Params>().modelName);
```

Comment:
question: why is decodeURIComponent necessary?

Reply:
A few characters (I know &, =, and /, probably ? and #) are not decoded in the param import. I get this error on the model page:

<img width="304" alt="Screen Shot 2022-01-25 at 1 25 26 PM" src="https://user-images.githubusercontent.com/643918/151045415-a4758f7e-571e-4b4d-87ae-16dfb03baeb6.png">

I'm banning / from model names because (even escaped) it messes up `/models/:name/versions`, but I don't want to find and ban all of the characters in this set.